pkg/ddl/job_scheduler.go

```diff
@@ -697,50 +697,69 @@ func insertDDLJobs2Table(ctx context.Context, se *sess.Session, jobWs ...*JobWra
 	return errors.Trace(err)
 }
 
-func job2SchemaIDs(jobW *JobWrapper) string {
-	return job2UniqueIDs(jobW, true)
-}
+func makeStringForIDs(ids []int64) string {
+	set := make(map[int64]struct{}, len(ids))
+	for _, id := range ids {
+		set[id] = struct{}{}
+	}
 
-func job2TableIDs(jobW *JobWrapper) string {
-	return job2UniqueIDs(jobW, false)
+	s := make([]string, 0, len(set))
+	for id := range set {
+		s = append(s, strconv.FormatInt(id, 10))
+	}
+	slices.Sort(s)
+	return strings.Join(s, ",")
 }
 
-func job2UniqueIDs(jobW *JobWrapper, schema bool) string {
+func job2SchemaIDs(jobW *JobWrapper) string {
 	switch jobW.Type {
-	case model.ActionExchangeTablePartition, model.ActionRenameTables, model.ActionRenameTable:
+	case model.ActionRenameTables:
```

Comment:
get the args from jobW.jobargs.(*model.RenameTablesArgs)

Reply:
no need to check version here, we have set jobW.JobArgs

below too